ui/ui.js

```diff
@@ -217,10 +217,17 @@ shaka.ui.Overlay = class {
         'manifestPeriodCount',
         'manifestGapCount',
       ],
+      adStatisticsList: [
+        'loadTimes',
+        'started',
+        'playedCompletely',
+        'skipped',
+      ],
       contextMenuElements: [
         'loop',
         'picture_in_picture',
         'statistics',
+        'ad_statistics',
```

Comment:
Be sure to add this to docs/tutorials/ui-customization.md

Reply:
Done!

Reply:
That doc also lists elements that can be added to different UI areas. See, for example, where it lists `picture_in_picture`

Reply:
Done!